docs/install.md

```diff
@@ -826,27 +826,164 @@ INFO: deploying from /nvme/gopath/src/github.com/intel/pmem-csi/deploy/kubernete
 ...
 ```
 
-<!-- FILL TEMPLATE:
+### Metrics support
+
+:bangbang: | Metric support is an alpha feature. What data is provided may change.
```

Comment:
This works on GitHub (not surpising, I got this fromhttps://github.com/github/markup/issues/887) but not in Sphinx (just shows `:bangbang:`):

https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/job/pmem-csi/job/PR-715/Doc_20Site/docs/install.html#metrics-support

@intelkevinputnam : do you have a suggestion how we can mark up warnings in our documentation?

Reply:
I'll merge without this warning and instead moved it into an issue: https://github.com/intel/pmem-csi/issues/736